synthesis/codon/example_test.go

```diff
@@ -94,52 +68,29 @@ func ExampleWriteCodonJSON() {
 	// cleaning up test data
 	os.Remove("../../data/codon_test.json")
 
-	fmt.Println(testCodonTable.GetAminoAcids()[0].Codons[0].Weight)
+	fmt.Println(testCodonTable.GetWeightedAminoAcids()[0].Codons[0].Weight)
 	//output: 28327
 }
 
 func ExampleCompromiseCodonTable() {
 	sequence, _ := genbank.Read("../../data/puc19.gbk")
-	codonTable := codon.GetCodonTable(11)
-
-	// a string builder to build a single concatenated string of all coding regions
-	var codingRegionsBuilder strings.Builder
-
-	// iterate through the features of the genbank file and if the feature is a coding region, append the sequence to the string builder
```

Comment:
this is needed.

Reply:
Needed for what? The functionality is all there. Is it readability or something you're concerned about?

Reply:
Just doubled checked. Functionality got moved to a method so it's fine.